corehq/apps/reports/standard/deployments.py

```diff
@@ -139,6 +141,22 @@ def rows(self):
             )
         return rows
 
+    @property
+    def export_table(self):
+        def _fmt_ordinal(ordinal):
+            if ordinal >= 0:
+                return safe_strftime(date.fromordinal(ordinal), USER_DATE_FORMAT)
+            return 'Never'
```

Comment:
Good point, perhaps this should be consistent with other exports instead of with the web version. "---" seems most consistent with other exports.